core/templates/services/auth.service.ts

```diff
@@ -27,25 +27,25 @@ import { AppConstants } from 'app.constants';
 import { AuthBackendApiService } from 'services/auth-backend-api.service';
 
 abstract class AuthServiceImpl {
-  abstract getRedirectResultAsync(): Promise<firebase.auth.UserCredential>;
-  abstract signInWithRedirectAsync(): Promise<void>;
-  abstract signOutAsync(): Promise<void>;
+   abstract getRedirectResultAsync(): Promise<firebase.auth.UserCredential>;
```

Comment:
Why these changes? (Ditto elsewhere in this file)

Reply:
Reverted!

Reply:
I have done the changes. PTAL.